notebooks/README.md

```diff
@@ -12,3 +12,4 @@
 * [Hugging Face Vision Transformer](https://nbviewer.org/github/skorch-dev/skorch/blob/master/notebooks/Hugging_Face_VisionTransformer.ipynb)
 * [Skorch Doctor](https://nbviewer.org/github/skorch-dev/skorch/blob/master/notebooks/Skorch_Doctor.ipynb)
 * [Classification with LLMs](https://nbviewer.org/github/skorch-dev/skorch/blob/master/notebooks/LLM_Classifier.ipynb)
+* [Learning Rate Scheduler]()
```

Comment:
This link can already be prefilled, right?

Reply:
I have updated the missing link.